### PR TITLE
fix for HF model restoration

### PR DESCRIPTION
**Pull request by ekmb**
Signed-off-by: ekmb <ebakhturina@nvidia.com>

### CODE REVIEW

File: nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py [hunk 1]
```diff
@@ -64,6 +64,13 @@
     },
 }
 
+VOCAB_FILE_NAME = {
```
Comment (ericharper) [inline nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py:67]: Can this be avoided? is it possible to return the path with `save_vocabulary`?

Comment (ekmb) [reply nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py:67]: The different models mean different things by "vocab_file", compare: https://github.com/huggingface/transformers/blob/master/src/transformers/models/albert/tokenization_albert.py#L33 vs https://github.com/huggingface/transformers/blob/master/src/transformers/models/bert/tokenization_bert.py#L32. I think it's better to preserve this info for the users

Comment (ericharper) [reply nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py:67]: It's fine to preserve it, but can we add the functionality to save_vocabulary. Also, what happens if the tokenizer_name is not in this dictionary?

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**